actions/save-dependabot-changes/action.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Save dependabot changes'
+description: "Check if dependencies were changed by dependabot and commit the new dependency information."
+inputs:
+  github_token:
+    description: |
+      Token used to commit changes back into the repository. 
+      This should be a personal access token, or commits made by this action won't trigger a new workflow run.
+      See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow for more info.
```

Comment:
If the actions are committing back to the same repo that is running the action does the default token have high enough permissions? We might be able to get away with just using the `GITHUB_TOKEN` that is baked into the pipeline 🤔

Reply:
@AidanWenzel this is fixed in PR [48](https://github.com/datawire/go-mkopensource/pull/48). I didn't want to make this one too big and since it had been in review for some time I decided to update ths in a new PR.